CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 ### Breaking
 ### Features
 ### Improvements
+
+* add `CONTRIBUTING` file
```

Comment:
The name is suggested by github and common in OSS Repos

Reply:
Do you have examples? I've skimmed through a couple and they all use `.md` or `.rst`.
The examples on the [github guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#examples-of-contribution-guidelines) use `.md` as well.

Reply:
<img width="745" alt="image" src="https://github.com/user-attachments/assets/025d4ec0-dc5f-4e2e-a614-3b1db552d10d">

I marked it in your suggested links. The file really is called `CONTRIBUTING` 
And the `LICENSE` file is markdown too, but not suffixed with `.md`

Reply:
I've seen that. However, none of the examples from that page do it like that. I've found no project on github that does not use `md` or `rst`.
Also, it states `... type the name and extension for the file. ... Files are rendered in rich text format if the file extension is in a supported format.`.
It just seems to me that they don't use an extension in the text, since it depends on the content of your CONTRIBUTING file.
Have a look at the most stared github projects, they usually have a `CONTRIBUTING.md` file.

The LICENSE file doesn't need to be rendered so it has no extension.

Reply:
I'm fine with that. I renamed the file. Lets try.

Reply:
Thank you.